src/_includes/layouts/pattern-library.hbs

```diff
@@ -4,43 +4,85 @@ styles:
 - /styles/eleventy-patterns/index
 scripts:
 - /scripts/eleventy-patterns/index
+- /scripts/eleventy-patterns/navigation
 ---
 <div class="EP_layout">
-  <aside class="EP_layout__sidebar">
-    <nav class="EP_nav">
+  <div class="EP_header-container">
+    <header class="EP_header">
       <h2>
-        <a href="/">
+        <a href="/" class="EP_nav__logo">
           Eleventy Patterns
         </a>
       </h2>
-
-      <div class="EP_nav__category">
-        <h3>
-          <a href="/prototypes">
+      <button class="EP_nav__menu-toggle EP_button" aria-expanded="false">Menu</button>
```

Comment:
I forget how `aria-expanded` targets the expanded collapsed area. Can you point to an ID or something for some screen readers?